src/colab/Run_SKAI_Colab_Pipeline.ipynb

```diff
@@ -337,11 +347,34 @@
         "#@title Generate Examples\n",
         "#@markdown First, generate the building images. This task should take about 30-45 minutes.\n",
         "\n",
+        "#@markdown Select if you want to generate your example from a pre-labeled file or generate unlabeled examples.\n",
+        "GENERATING_JOB = \"unlabeled\" #@param [\"pre_labeled\",\"unlabeled\"]\n",
+        "\n",
         "## COMMAND RUN\n",
         "timestamp = datetime.datetime.now().strftime('%Y%m%d%H%M%S')\n",
         "GCP_DATASET_NAME = f\"{Author}_example_{timestamp}_{PROJECT_DIRECTORY}\".replace(\"_\",\"-\").lower()\n",
         "\n",
-        "generate_examples_args = {\n",
+        "if GENERATING_JOB == \"pre_labeled\":\n",
```

Comment:
Can we change "pre_labeled" to "labeled" so that we stick with one term?

Reply:
Thanks it makes sense. I will aplly the change.